djoser/utils.py

```diff
@@ -8,6 +8,16 @@
 except ImportError:
     from django.contrib.sites.models import get_current_site
 
+try:
+    from django.utils.module_loading import import_string
+except ImportError:
+    from django.utils.module_loading import import_by_path
```

Comment:
`from django.utils.module_loading import import_by_path as import_string` will be simpler, than another function